src/plugins/discover/public/application/angular/context/api/anchor.ts

```diff
@@ -6,11 +6,27 @@
  * Side Public License, v 1.
  */
 
-import _ from 'lodash';
+import _, { Dictionary } from 'lodash';
```

Comment:
while cleaning this up, could you migrate the use of `_` to just import `get` from lodash? many thx!